test/parquet/prefetching.test

```diff
@@ -0,0 +1,35 @@
+# name: test/parquet/prefetching.test
+# description: Test parquet files using the prefetching mechanism
+# group: [parquet]
+
+require parquet
+
+# Normally, local files do not use the prefetching mechanism, however this debugging options will force the mechanism
+statement ok
+set prefetch_all_parquet_files=true;
+
+# With default settings, this query will fail: the incorrectly set index page offsets mess with duckdb's prefetching mechanism
+statement error
+FROM 'data/parquet-testing/incorrect_index_page_offsets.parquet'
+----
+IO Error: The parquet file 'data/parquet-testing/incorrect_index_page_offsets.parquet' seems to have incorrectly set page offsets. This interferes with DuckDB's prefetching optimization. DuckDB may still be able to scan this file by manually disabling the prefetching mechanism using: 'SET disable_parquet_prefetching=true'.
+
+# Now we disable prefetching
+statement ok
+set disable_parquet_prefetching=true;
+
+# This file messes with DuckDB's prefetching mechanism, however the prefetching mechanism automatically disables
+# when it detects this situation meaning that the query still succeeds (possibly at a performance penalty) 
```

Comment:
Outdated? Also the title of the PR I think

Reply:
indeed, thx!